web/README.md

```diff
@@ -4,10 +4,12 @@
 - Run `node --version` to make sure that you have at least Node.js 18 or above. If you are on **Ubuntu <= 22.04**, you
   need to
   [upgrade](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions).
-- Run `cd mitmproxy/web` to change to the directory with package.json
-- Run `npm install` to install dependencies
-- Run `npm start` to start live-compilation
-- Run `mitmweb` after activating your Python virtualenv (see [`../CONTRIBUTING.md`](../CONTRIBUTING.md)).
+- Run `cd ./web` to change to the directory with package.json.
+- Run `npm install` to install dependencies.
+
+
+- Run `uv run mitmweb`
```

Comment:
Nitpick: this looks weird in text format and has no visual implications when rendered on GitHub.

```suggestion
- Run `npm install` to install dependencies.
- Run `uv run mitmweb`
```